chef/cookbooks/cpe_utils/README.md

```diff
@@ -78,10 +78,61 @@ remote_pkg 'VMWareTools' do
 end
 ```
 
+#### node.virtual_macos_type ####
+This is similar to node.virtual?, except it will allow you to apply granular conditions based on virtual machine type. This is useful for scoping tools to specific virtualization platforms.
+
+```
+def parallels?
```

Comment:
Could these shortcut functions be added to node_functions?

Then the readme would be simplified to an example of gating something to parallels, for instance:

```
remote_pkg 'parallels_tools'
  only_if { node.parallels? }
  source 'https://someserver/sometool.pkg'
end
```